docs/source/user-guide/tabular/tutorials/plot_static_predictions.py

```diff
@@ -0,0 +1,82 @@
+# -*- coding: utf-8 -*-
+"""
+==============================
+Using Pre-computed Predictions
+==============================
+
+There are several cases in which it's useful to compute the model predictions before using deepchecks.
+
+In some cases the model evaluation can take a long time i.e very large dataset.
+This feature can also be helpful if the Model inference happens on a production node and you can get the predictions
+using api.
+You can also use this feature to run deepchecks on models that are not sklearn compatible.
```

Comment:
```suggestion
You can also use this feature to run deepchecks on models that do not support the sklearn api.
```
And link here to "supported models"